Disable adjustment and hyphenation (local change).
Set .TH and .MR in roman, not italic font (local fix for 1.23 regression).
Map hyphen-minus and accents to ASCII in the traditional way (local change).
Index: tmac/man.local
--- tmac/man.local.orig
+++ tmac/man.local
@@ -28,6 +28,31 @@
 .\" Set this register to configure which the `MR` macro uses.
 .\" .nr an*MR-URL-format 1
 .\"
+.\" ----------------------------------------------------------------------
+.\" OpenBSD-specific changes
+.\" ----------------------------------------------------------------------
+.\" Disable adjustment and hyphenation by default,
+.\" such that manuals look similar with groff and mandoc(1).
+.ad l
+.de ad
+..
+.nh
+.de hy
+.nh
+..
+.\" Fix regression: do not use italic font in .TH and .MR
+.ds MF R
+.\"
+.\" Map ASCII input single quotes and hyphen-minus to ASCII output
+.\" until we properly escape them in OpenBSD manual pages.
+.if '\*[.T]'utf8' \{\
+.  char ' \N'39'
+.  char - \N'45'
+.  char ^ \N'94'
+.  char ` \N'96'
+.  char ~ \N'126'
+.\}
+.\"
 .\" Local Variables:
 .\" mode: nroff
 .\" fill-column: 72
